pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -55,6 +55,9 @@ func NewHealthChecker(monCluster *Cluster) *HealthChecker {
 
 	monCRDTimeoutSetting := monCluster.spec.HealthCheck.DaemonHealth.Monitor.Timeout
 	if monCRDTimeoutSetting != "" {
+		if monCRDTimeoutSetting == "0" {
+			logger.Warning("monitor failover is disabled")
```

Comment:
How about info? 
```suggestion
			logger.Info("monitor failover is disabled")
```

Reply:
I feel like this is not an optimal situation thus "warning" seems more appropriate IMO.